sys/net/application_layer/gcoap/forward_proxy.c

```diff
@@ -250,7 +244,7 @@ static void _set_response_type(coap_pkt_t *pdu, uint8_t resp_type)
     }
 }
 
-static void _forward_resp_handler(const gcoap_request_memo_t *memo,
+void _forward_resp_handler(const gcoap_request_memo_t *memo,
```

Comment:
```suggestion
void _gcoap_forward_resp_handler(const gcoap_request_memo_t *memo,
```

Just to be on the save side naming-wise. ;-)